### PR TITLE
[New Rule] AWS ElastiCache Security Group Created

### LINKED ISSUE #1354: [New Rule] AWS ElastiCache Security Group Created
## Description
Identifies when an ElastiCache security group has been created.

### Target indexes

`filebeat-*, logs-aws*`

### Platforms

AWS Cloudtrail

### Query

```
event.dataset:aws.cloudtrail and  event.provider:elasticache.amazonaws.com and 
event.action:"Create Cache Security Group"  and  event.outcome:success
```

### **False Positives**
- 

### MITRE

**ATTACK TACTIC**
Credential Access, Persistence

**ATTACK TECHNIQUE**
Account Manipulation

### References
- https://docs.aws.amazon.com/AmazonElastiCache/latest/APIReference/Welcome.html

### PR DESCRIPTION
**Pull request by austinsonger**
## Issues

Resolves #1354

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: rules/integrations/aws/persistence_elasticache_security_group_creation.toml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+from = "now-60m"
+index = ["filebeat-*", "logs-aws*"]
+interval = "10m"
+language = "kuery"
+license = "Elastic License v2"
+name = "AWS ElastiCache Security Group Created"
+note = """## Config
+
+The AWS Fleet integration, Filebeat module, or similarly structured data is required to be compatible with this rule."""
+references = ["https://docs.aws.amazon.com/AmazonElastiCache/latest/APIReference/Welcome.html"]
```
Comment (w0rk3r) [inline rules/integrations/aws/persistence_elasticache_security_group_creation.toml:25]:
```suggestion
references = ["https://docs.aws.amazon.com/AmazonElastiCache/latest/APIReference/API_CreateCacheSecurityGroup.html"]
```

File: rules/integrations/aws/persistence_elasticache_security_group_creation.toml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+[[rule.threat.technique.subtechnique]]
+id = "T1136.003"
+name = "Cloud Account"
+reference = "https://attack.mitre.org/techniques/T1136/003/"
+
+
+[rule.threat.tactic]
+id = "TA0003"
+name = "Persistence"
+reference = "https://attack.mitre.org/tactics/TA0003/"
```
Comment (w0rk3r) [inline rules/integrations/aws/persistence_elasticache_security_group_creation.toml:54]:
```suggestion
[[rule.threat.technique]]
id = "T1562"
name = "Impair Defenses"
reference = "https://attack.mitre.org/techniques/T1562/"

  [[rule.threat.technique.subtechnique]]
  id = "T1562.007"
  name = "Disable or Modify Cloud Firewall"
  reference = "https://attack.mitre.org/techniques/T1562/007/"

[rule.threat.tactic]
name = "Defense Evasion"
id = "TA0005"
reference = "https://attack.mitre.org/tactics/TA0005/"
```

This one fits better in this case, also we need to rename the rule to `defense_evasion_elasticache_security_group_creation.toml`

**Review by w0rk3r — CHANGES_REQUESTED** [review]
Hey @austinsonger, thanks for the contribution. Here are some required changes before we merge this one

File: rules/integrations/aws/persistence_elasticache_security_group_creation.toml (hunk 2 above)
Comment (w0rk3r) [reply rules/integrations/aws/persistence_elasticache_security_group_creation.toml:54]: This one still needs to be fixed/discussed.

**Review by w0rk3r — APPROVED** [review]
LGTM

File: rules/integrations/aws/defense_evasion_elasticache_security_group_creation.toml [hunk 3]
```diff
@@ -0,0 +1,53 @@
+[metadata]
+creation_date = "2021/07/19"
+maturity = "production"
+updated_date = "2021/10/01"
```
Comment (w0rk3r) [inline rules/integrations/aws/defense_evasion_elasticache_security_group_creation.toml:4]:
```suggestion
updated_date = "2021/10/01"
integration = "aws"
```

**Review by w0rk3r — CHANGES_REQUESTED** [review]

File: rules/integrations/aws/defense_evasion_elasticache_security_group_creation.toml (hunk 3 above)
Comment (w0rk3r) [reply rules/integrations/aws/defense_evasion_elasticache_security_group_creation.toml:4]: Missed this one, after this, LGTM

**Review by w0rk3r — APPROVED** [review]
Thanks! LGTM

File: rules/integrations/aws/defense_evasion_elasticache_security_group_creation.toml [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+references = ["https://docs.aws.amazon.com/AmazonElastiCache/latest/APIReference/API_CreateCacheSecurityGroup.html"]
+risk_score = 21
+rule_id = "7b3da11a-60a2-412e-8aa7-011e1eb9ed47"
+severity = "low"
+tags = ["Elastic", "Cloud", "AWS", "Continuous Monitoring", "SecOps", "Monitoring"]
+timestamp_override = "event.ingested"
+type = "query"
+
+query = '''
+event.dataset:aws.cloudtrail and  event.provider:elasticache.amazonaws.com and event.action:"Create Cache Security Group" and
```
Comment (brokensound77) [inline rules/integrations/aws/defense_evasion_elasticache_security_group_creation.toml:35]:
```suggestion
event.dataset:aws.cloudtrail and event.provider:elasticache.amazonaws.com and event.action:"Create Cache Security Group" and 
```

**Review by brokensound77 — APPROVED** [review]
👍

**Pull request merged**